lint_clean_files.sh

```diff
@@ -1,7 +1,16 @@
 #!/usr/bin/env bash
 
 # Pull list of checkable files from clean_files.txt
+#  - Folder references are allowed
+#  - Empty lines are ignored
+#  - Comment lines (#) are ignored
 #
-mapfile -t FILES < clean_files.txt
+# shellcheck disable=SC2002  # Prefer 'cat' for cleaner script
+mapfile -t FILES < <(
+	cat clean_files.txt \
+		| grep -v -E '^\s*$' \
+		| grep -v -E '^\s*#' \
+		| xargs -n1 -I{} find "{}" -type f
+)
 
-pre-commit run --files "${FILES[@]}"
+BASH_IT='' pre-commit run --files "${FILES[@]}"
```

Comment:
You should add a comment here explaining why you do this

Reply:
Done, good suggestion, thanks !